metrics/web/lib/dashboard/presentation/widgets/no_search_results_placeholder.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/base/presentation/widgets/decorated_container.dart';
+import 'package:metrics/common/presentation/metrics_theme/widgets/metrics_theme.dart';
+import 'package:metrics/common/presentation/text_placeholder/widgets/text_placeholder.dart';
+import 'package:metrics/dashboard/presentation/strings/dashboard_strings.dart';
+
+/// A widget that displays a [DashboardStrings.noSearchResults] text inside a bordered container.
+class NoSearchResultsPlaceholder extends StatelessWidget {
+  const NoSearchResultsPlaceholder({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = MetricsTheme.of(context)
+        .projectMetricsTableTheme
+        .projectMetricsTileTheme;
+
+    return DecoratedContainer(
+      width: 1140.0,
```

Comment:
It seems like we don't actually need to set the width here since this widget will be expanded to fill all available space on the screen because it is wrapped in the `Expanded` widget.